warehouse/models/mart/gtfs/fct_monthly_shapes.sql

```diff
@@ -0,0 +1,82 @@
+{{ config(materialized='table') }}
+
+WITH dim_gtfs_datasets AS (
+    SELECT *
+    FROM {{ ref('dim_gtfs_datasets') }}
+),
+
+fct_scheduled_trips AS (
+    SELECT *
+    FROM {{ ref('fct_scheduled_trips') }}
+    WHERE service_date >= '2023-06-01'
+),
+
```

Comment:
I'd add `fct_daily_schedule_feeds` here to help with the join with dim shapes array below